src/spawnUnits/main.ts

```diff
@@ -50,7 +50,8 @@ import {
   insertOrUpdateSpawnGroup,
   typeNamesFrom,
 } from '../db/spawnGroups'
-import { Country } from '../../generated/dcs/common/v0/Country'
+
+const CONST_DEFAULT_DESTROY_RADIUS_METERS = 250
```

Comment:
```suggestion
const DEFAULT_DESTROY_RADIUS_METERS = 250
```

Reply:
Is this name slightly misleading?

there is no default "destroy radius" as by default destroy only destroys one thing. there is a "search radius" for nearby "thing to destroy"
but, technical, no destroy radius default.